app/views/standard/addClosureMetadata.scala.html

```diff
@@ -11,94 +11,99 @@
-                                            Symbol("_dynamicForm") -> true
-                                        ),
-                                        fieldErrors
-                                    )
-                            }
-                        }
-                    }
-                    <div class="govuk-button-group">
-                            <!-- Continue -->
-                        <button data-prevent-double-click="true" class="govuk-button" type="submit" data-module="govuk-button" role="button">
-                            Continue
-                        </button>
-                            <!-- Redirects to Homepage for now but will probably just take you back to previous page -->
-                        <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel</a>
-                    </div>
-                </div>
+                case DateField(fieldId, fieldName, fieldDescription, day, month, year, _, fieldErrors, _) => inputDate(
+                  fieldId,
+                  fieldName,
+                  fieldDescription,
+                  day,
+                  month,
+                  year,
+                  fieldErrors,
+                  Map(Symbol("_dynamicForm") -> true)
+                )
+                case TextField(fieldId, fieldName, fieldDescription, inputNameAndValue, inputMode, _, fieldErrors) => inputText(
+                  fieldId,
+                  fieldName,
+                  fieldDescription,
+                  inputNameAndValue,
+                  "number",
+                  fieldErrors,
+                  Map(Symbol("_dynamicForm") -> true),
+                  inputMode,
+                  addSuffixText = true
+                )
+                case DropdownField(fieldId, fieldName, fieldDescription, options, selectedOption, _, fieldErrors) =>
+                  //could theoretically be a checkbox or radio (since radios are not always boolean)
+                  inputDropdown(
+                    fieldId,
+                    fieldName,
+                    fieldDescription,
+                    options,
+                    selectedOption,
+                    Map(
+                      Symbol("_disabledOption") -> "",
+                      Symbol("_dynamicForm") -> true
+                    ),
+                    fieldErrors
+                  )
+              }
             }
-            @transferReference(consignmentRef, isJudgmentUser = false)
-            </div>
-
+          }
+          <div class="govuk-button-group">
+              <!-- Continue -->
+            <button data-prevent-double-click="true" class="govuk-button" type="submit" data-module="govuk-button" role="button">
+              Continue
+            </button>
+              <!-- Redirects to Homepage for now but will probably just take you back to previous page -->
+            <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel</a>
```

Comment:
Should take the user back to the navigation page

Reply:
Also, clicking the cancel button after I updated it caused this bug https://national-archives.atlassian.net/browse/TDR-2374 so I've fixed that as part of this PR.